manual/photon-dev.xml.template

```diff
@@ -30,6 +30,27 @@
             <Label>DNS Domain</Label>
             <Description>DNS Domain</Description>
         </Property>
+        <Category>Proxy Settings (optional)</Category>
+        <Property ovf:key="guestinfo.http_proxy" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+            <Label>HTTP Proxy</Label>
+            <Description>Enter HTTP Proxy Server followed by the port and without typing "http://" before. Example: "proxy.provider.de:3128"</Description>
+        </Property>
+        <Property ovf:key="guestinfo.https_proxy" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+            <Label>HTTPS Proxy</Label>
+            <Description>Enter HTTPS Proxy Server followed by the port and without typing "https://" before. Example: "proxy.provider.de:3128"</Description>
+        </Property>
+        <Property ovf:key="guestinfo.proxy_username" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+            <Label>Proxy Username (optional)</Label>
+            <Description>Username for the Proxy Server</Description>
+        </Property>
+        <Property ovf:key="guestinfo.proxy_password" ovf:password="true" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+            <Label>Proxy Password (optional)</Label>
+            <Description>Password for the Proxy User</Description>
+        </Property>
+            <Property ovf:key="guestinfo.no_proxy" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+            <Label>No Proxy</Label>
+            <Description>No Proxy for e.g. your internal domain suffix. Comma seperated (localhost, 127.0.0.1, domain.local)</Description>
```

Comment:
`seperated` is miss-spelled, it should be `separated`

Reply:
- [x] Fixed